jupyterhub/templates/hub/pvc.yaml

```diff
@@ -5,6 +5,7 @@ metadata:
   name: hub-db-dir
   labels:
     {{- include "jupyterhub.labels" . | nindent 4 }}
+    hub.jupyter.org/storage-kind: "core"
```

Comment:
Note that this can be applied on the PVC without trouble. You may not change the selector field in the spec of the PVC though.

Reply:
What does this storage-kind label do?

The extra labels makes sense, but I don't see what is accomplished by the 'storage-kind' label.

Reply:
The main benefit as I see it, is that having such labels allows us to write easier instructions for various tasks in the future, or simply allow someone to maintain a jupyterhub better thanks to `--selector`.

What would you do if you wanted to delete all PVCs, but NOT the hub-db-dir ? Well enter regexp and scripting currently, but if you have a label on the hub-db-dir you can exclude that with a `--selector`.

So, it is on other words a harmless but potentially useful thing, perhaps it can be useful to us if we support someone solve something regarding this as well.

Reply:
If there's a problem that user volumes aren't well labeled, we can address that by making sure they have a good label (is component not an adequate standard label that exists for this purpose?), but I think the kind=core/user distinction is being applied too broadly, and the core label isn't useful here, especially since it's being applied to exactly one already-distinguishible volume.

Reply:
Ah, the ideas of this were introduced before some other labeling improvements, now I see we have the following labels, which i find to be enough:

```yaml
  # on user PVC
  labels:
    app: jupyterhub
    chart: jupyterhub-0.7.0
    component: singleuser-storage
    heritage: jupyterhub
    release: jhub

  # on hub-db-dir PVC
  labels:
    app: jupyterhub
    chart: jupyterhub-0.7.0
    component: hub
    heritage: Tiller
    release: jhub
```

I'll delete the storage-kind stuff.